simulator/src/main/java/kcl/teamIndexZero/traffic/simulator/data/descriptors/TrafficLightsInSetRule.java

```diff
@@ -1,7 +1,24 @@
 package kcl.teamIndexZero.traffic.simulator.data.descriptors;
 
+import kcl.teamIndexZero.traffic.simulator.data.trafficLight.TrafficLight;
+import kcl.teamIndexZero.traffic.simulator.data.trafficLight.TrafficLightInSet;
+import kcl.teamIndexZero.traffic.simulator.data.trafficLight.TrafficLightState;
+
+import java.util.List;
+
 /**
  * Traffic Light in synchronous set rule
  */
 public class TrafficLightsInSetRule {
```

Comment:
So the traffic light state change logic is in a class that is supposed to be used as descriptor. How can we create synchronous (working together) sets of lights if this is here?